tests/jlm/llvm/frontend/llvm/StoreTests.cpp

```diff
@@ -25,7 +25,7 @@ StoreConversion()
   std::unique_ptr<Module> llvmModule(new Module("module", context));
 
   auto int64Type = Type::getInt64Ty(context);
-  auto pointerType = Type::getInt64PtrTy(context);
+  auto pointerType = llvm::PointerType::getUnqual(Type::getInt64Ty(context));
```

Comment:
I think the `PointerType::getUnqual()` without the type argument(!) is the right one to use here: https://llvm.org/doxygen/classllvm_1_1PointerType.html